core/tests/protractor_utils/forms.js

```diff
@@ -633,8 +635,9 @@ var CodeMirrorChecker = function(elem) {
       // codemirror has loaded. The (2i)th line contains a line number and the
       // (2i+1)th line contains the text on that line.
       var textArray = text.split('\n');
+      var lineNumber = null;
       for (var i = 0; i < textArray.length; i += 2) {
-        var lineNumber = textArray[i];
+        lineNumber = textArray[i];
```

Comment:
It seems to me like lineNumber here and lineNumber in line 655 are different things, and should therefore have different variable names...?

Reply:
Done.